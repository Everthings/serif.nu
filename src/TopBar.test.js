import React from 'react';
import renderer from 'react-test-renderer';
import TopBar from './TopBar';

it('renders correctly', () => {
  const tree = renderer.create(<TopBar />).toJSON();

  expect(tree).toMatchSnapshot();
});
