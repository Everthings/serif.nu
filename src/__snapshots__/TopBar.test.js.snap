// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<header
  className="MuiPaper-root-11 MuiPaper-elevation4-17 MuiAppBar-root-3 MuiAppBar-positionStatic-7 MuiAppBar-colorPrimary-9"
>
  <div
    className="MuiToolbar-root-38 MuiToolbar-regular-40 MuiToolbar-gutters-39"
  >
    <button
      aria-label="Navigation"
      className="MuiButtonBase-root-48 MuiIconButton-root-42"
      disabled={false}
      onBlur={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchMove={[Function]}
      onTouchStart={[Function]}
      tabIndex="0"
      type="button"
    >
      <span
        className="MuiIconButton-label-47"
      >
        <svg
          aria-hidden="true"
          className="MuiSvgIcon-root-51 TopBar-icon-2"
          focusable="false"
          viewBox="0 0 24 24"
        >
          <path
            d="M0 0h24v24H0z"
            fill="none"
          />
          <path
            d="M3 18h18v-2H3v2zm0-5h18v-2H3v2zm0-7v2h18V6H3z"
          />
        </svg>
      </span>
      <span
        className="MuiTouchRipple-root-58"
      />
    </button>
    <h2
      className="MuiTypography-root-65 MuiTypography-title-71 TopBar-title-1"
    >
      Serif.nu
    </h2>
  </div>
</header>
`;
